### PR TITLE
[i18n] add German transaltions

### PR DESCRIPTION
**Pull request by Nokius**
I hope that the translation is clear to others. 

Hopefully I added no typo :-S

Thanks for the great application!

### CODE REVIEW

**Comment by aebruno** [comment]
Thank you!!

File: qml/i18n/whisperfish_de.ts [hunk 1]
```diff
@@ -91,121 +91,121 @@
...
         <extracomment>Add Device</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Gerät hinzufügen</translation>
     </message>
     <message id="whisperfish-device-link-instructions">
         <location filename="../pages/AddDevice.qml" line="52"/>
         <source>Install Signal Desktop. Use the CodeReader application to scan the QR code displayed on Signal Desktop and copy and paste the URL here.</source>
         <extracomment>Instructions on how to scan QR code for device linking</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Installiere Signal Desktop. Nutze eine QR-Code Scanner um die URL aus dem QR-Code hinzuzufügen.</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: Isn't it "Nutze eine**n** QR-Code Scanner..." ?

File: qml/i18n/whisperfish_de.ts [hunk 2]
```diff
@@ -91,121 +91,121 @@
...
         <extracomment>Linked device date</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Verbunden: %1</translation>
     </message>
     <message id="whisperfish-device-last-active">
         <location filename="../pages/LinkedDevices.qml" line="94"/>
         <source>Last active: %1</source>
         <extracomment>Linked device last active date</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Letzte aktivität: %1</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: I guess its "Letzte **A**ktivität"

File: qml/i18n/whisperfish_de.ts [hunk 3]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Reset secure session menu item</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Sichere Sitzung zurücksetzen</translation>
     </message>
     <message id="whisperfish-reset-session-message">
         <location filename="../pages/VerifyIdentity.qml" line="23"/>
         <source>Resetting secure session</source>
         <extracomment>Reset secure session remorse message</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Sichere Sitzung wird zurücksetzen</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: Isn't it "...wird zurück**gesetzt**"?

File: qml/i18n/whisperfish_de.ts [hunk 4]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Phone number input</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Rufnummer (im E.164 Format</translation>
     </message>
     <message id="whisperfish-phone-number-input-placeholder">
         <location filename="../pages/Register.qml" line="46"/>
         <source>18875550100</source>
         <extracomment>Phone number placeholder</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>4901751234567</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: Sehr gute Idee! Wie das richtige format ist musste ich auch erst ein paar mal ausprobieren.

File: qml/i18n/whisperfish_de.ts [hunk 5]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Settings page notifications show message body</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Zeige Nachrichteninhalt</translation>
     </message>
     <message id="whisperfish-settings-general-section">
         <location filename="../pages/Settings.qml" line="117"/>
         <source>General</source>
         <extracomment>Settings page general section</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>ALlgemein</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: There is capital L

File: qml/i18n/whisperfish_de.ts [hunk 6]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Settings page stats section</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Statistiken</translation>
     </message>
     <message id="whisperfish-settings-websocket">
         <location filename="../pages/Settings.qml" line="193"/>
         <source>Websocket Status</source>
         <extracomment>Settings page websocket status</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Websocket status</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: Status

File: qml/i18n/whisperfish_de.ts [hunk 7]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Verify Password label</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Passwort überprüfen</translation>
     </message>
     <message id="whisperfish-password-info">
         <location filename="../pages/Password.qml" line="97"/>
         <source>Whisperfish stores identity keys, session state, and local message data encrypted on disk. The password you set is not stored anywhere and you will not be able to restore your data if you lose your password. Note: Attachments are currently stored unencrypted. You can disable storing of attachments in the Settings page.</source>
         <extracomment>Whisperfish password informational message</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Whisperfish speichert Identitätsschlüssel, Sitzungsverläufe und locale Nachrichten verschlüsselt ab. Das Passwort wird nicht gespeichert, ein Verlust dessen bedeuted, dass Du die Daten nicht wiederherstellen kannst. Beachte: Anhänge werden derzeit unverschlüsselt gespeichert. Das Speichern von Anhängen kann in den Einstellungen deaktiviert werden.</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: lokale

File: qml/i18n/whisperfish_de.ts [hunk 8]
```diff
@@ -217,458 +217,458 @@
...
         <extracomment>Reset secure session remorse message</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Sichere Sitzung wird zurücksetzen</translation>
     </message>
     <message id="whisperfish-numeric-fingerprint-directions">
         <location filename="../pages/VerifyIdentity.qml" line="64"/>
         <source>If you wish to verify the security of your end-to-end encryption with %1, compare the numbers above with the numbers on their device.</source>
         <extracomment>Numeric fingerprint instructions</extracomment>
-        <translation type="unfinished"></translation>
+        <translation>Um die Sicherheit, der End-zu-End verschlüsselung zu verifizieren mit %1, vergleiche die obrieben Nummer mit den auf dem Gerät des Gesprächspatner.</translation>
```
Comment (bonanza123) [inline qml/i18n/whisperfish_de.ts]: Ende-zu-Ende-Verschlüsselung

**Comment by bonanza123** [comment]
Thanks for the translations. I made some comments please check whether they are useful.

**Comment by Nokius** [comment]
@bonanza123 please review #67 Thanks

File: qml/i18n/whisperfish_de.ts (hunk 8 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 7 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 6 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 5 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 4 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: me too

File: qml/i18n/whisperfish_de.ts (hunk 3 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 2 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

File: qml/i18n/whisperfish_de.ts (hunk 1 above)
Comment (Nokius) [inline qml/i18n/whisperfish_de.ts]: +1

**Pull request merged**